opus/application/apps/tools/file_utils.py

```diff
@@ -87,8 +90,45 @@ def get_pds_products(opus_id_list,
     sql += ' FROM '+q('obs_files')
     sql += ' WHERE '
     if product_types != ['all']:
-        sql += q('obs_files')+'.'+q('short_name')+' IN %s AND '
-        values.append(product_types)
+        sql += '('
+        # Because we didn't store @version in the database, when multiple versions (or
+        # product types) are passed in, we need to query the database by both short name
+        # and version name for each product type in the product_types list. The query
+        # condition will be something like:
+        # (obs_files.short_name='coiss_calib' AND obs_files.version_name='Current') OR
+        # (obs_files.short_name='coiss_calib' AND obs_files.version_name='1') OR
+        # (obs_files.short_name='coiss_raw' AND obs_files.version_name='Current') OR
+        # ...
+        for i, p in enumerate(product_types):
+            # Uncomment this code if we would like to display all files when the user
+            # passes empty string to types parameter. Otherwise we assume the user is
+            # specifying empty string for the types.
+            # if not p:
+            #     continue
+
+            # Check and see if the product type has a version specified (look for '@')
+            if settings.FILE_VERSION_MODIFIER in p:
+                prod_type, _, version = p.partition(settings.FILE_VERSION_MODIFIER)
+                if version == 'current':
+                    version = 'Current'
+                else:
+                    # Support cases like @v1.0, @v1, @v1.2, @1.0, @1, and @1.2
+                    if 'v' in version:
+                        version = version[1:]
```

Comment:
This doesn't necessarily work, since the v could be somewhere else but then we always strip off the first character. How about
version = version.lstrip('v')